source/math/FloatRandomGenerator.ooc

```diff
@@ -65,12 +75,24 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 		this _backend free()
 		super()
 	}
+	setRange: func (=_mu, =_sigma) {
+		_hasSecond = false
```

Comment:
And who calls this function? Is there any reason someone would create a `FloatGaussianRandomGenerator`, use it for a while, and then set its range?